### DISCUSSION TITLE
Situation with multiple inverter support

### DISCUSSION THREAD

**Discussion by richjgould** [kellerza/sunsynk#166] [General]
Been using the addon for about a week with my Sunsynk 3.6kw inverter.
Absolutely fantastic and delighted to get this data out of the cloud and into HA in real time.
I'm considering getting a second sunsynk inverter running in parallel but nervous if it means I loose my ability to use this add-on!
What's the status of multiple inverter support?

thanks for all your efforts

Richard

**Comment by kellerza** [comment]
Multiple inverters have been supported for a while. With the latest release first inver config is supported and concurrent reads from all inverters. Docs mostly up to date
